### PR TITLE
Login module Updates

### PR DESCRIPTION
**Pull request by localastronaut**

### CODE REVIEW

**Comment by RevolutionTech** [comment]
Looks like this branch has conflicts. Can you merge master into this branch and resolve them?

File: static/css/perdiem.css [hunk 1]
```diff
@@ -253,3 +253,26 @@ a.nav-color {
...
+    text-align: center;
+    padding: 20px;
+}
+
+#login-modal label {
+    color: #fff;
+}
+
+form > input[type=submit] {
+    display:block;
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: Would like to see a space between attribute and value.
```css
form > input[type=submit] {
    display: block;
    margin: 0 auto;
}
```

**Comment by localastronaut** [comment]
Updated from master but don't see any conflicts

File: templates/base.html [hunk 2]
```diff
@@ -102,36 +102,40 @@
...
+            </div>
             <button class="close-button" data-close aria-label="Close modal" type="button">
                 <span aria-hidden="true">&times;</span>
             </button>
         </div>
 
         <!-- Error Modal -->
         <div class="reveal" id="error-modal" data-reveal>
-            <h1>Error</h1>
+            <h1 class="white-color center">Error</h1>
```
Comment (RevolutionTech) [inline templates/base.html]: If you're going to put `DONE` above in all caps, might as well make `ERROR` all caps for consistency.

File: static/css/perdiem.css [hunk 3]
```diff
@@ -271,3 +271,8 @@ a.nav-color {
 #login-modal label {
     color: #fff;
 }
+
+form > input[type=submit] {
+    display:block;
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: Would like to see a space between attribute and value.
```css
form > input[type=submit] {
    display: block;
    margin: 0 auto;
}
```

**Comment by RevolutionTech** [comment]
Sometimes git can actually resolve the conflicts automatically. It's just that in certain cases Github isn't smart enough to realize that.

File: static/css/perdiem.css [hunk 4]
```diff
@@ -272,7 +272,7 @@ a.nav-color {
     color: #fff;
 }
 
-form > input[type=submit] {
+form > input [type=submit] {
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: This change is not what I meant. For each CSS attribute for this selector, there is a attribute name and then colon and then value. There should be a space between the colon and the value. ie. it should look like:
```css
form > input[type=submit] {
    display: block;
    margin: 0 auto;
}
```

Comment (localastronaut) [inline static/css/perdiem.css]: Ahh. Yeah, I totally missed that my bad!

**Pull request merged**